### PR TITLE
[AppConfig] update version for Sep release

### PR DESCRIPTION
**Pull request by mssfang**
Update current version 1.2.0-beta.1 to 1.1.5 in preparation for September release

### CODE REVIEW

**Review by conniey — APPROVED** [review]

File: sdk/appconfiguration/azure-data-appconfiguration/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-data-appconfiguration</artifactId>
-  <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;current} -->
+  <version>1.1.5</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;current} -->
```
Comment (samvaity) [inline sdk/appconfiguration/azure-data-appconfiguration/pom.xml:16]: Still need to wait for Identity releases to be shipped.

**Review by samvaity — CHANGES_REQUESTED** [review]

**Comment by samvaity** [comment]
>/azp run java - textanalytics - tests

Did you intend to run app config tests instead?

**Review by alzimmermsft — APPROVED** [review]

**Review by samvaity — APPROVED** [review]

**Pull request merged**